pkg/model/manifest.go

```diff
@@ -474,6 +474,7 @@ func inferHelmTags(path string) string {
 		}
 		if image.Image != "" {
 			result += fmt.Sprintf(" --set %s.image=${OKTETO_BUILD_%s_IMAGE}", svcName, strings.ToUpper(svcName))
+			os.Setenv(fmt.Sprintf("OKTETO_BUILD_%s_IMAGE", strings.ToUpper(svcName)), image.Image)
```

Comment:
is this change related to this PR?

Reply:
not related, but it's a bug that needs to be fixed. When we deploy a helm chart without Okteto Manifest we were not setting that env var which was causing an error